public/src/index.ts

```diff
@@ -41,9 +40,7 @@ async function handleStockPricesStream() {
 
     for await (const stockPricesUpdate of stream) {
         const update = stockPricesUpdate.getPricesMap();
-        console.log("Stock prices update");
         update.forEach((newPrice, stockId) => {
```

Comment:
Remove the `forEach` loop as well, since the only statement in it has been removed.